src/status_im2/subs/chat/messages.cljs

```diff
@@ -102,63 +102,58 @@
    (keep #(if (= :message (% :type))
             (when-let [message (messages (% :message-id))]
               (let [pinned-message (get pinned-messages (% :message-id))
-                    pinned         (if pinned-message true (some? (message :pinned-by)))
-                    pinned-by      (when pinned (or (message :pinned-by) (pinned-message :pinned-by)))
-                    message        (assoc message :pinned pinned :pinned-by pinned-by)]
+                    pinned (if pinned-message true (some? (message :pinned-by)))
+                    pinned-by (when pinned (or (message :pinned-by) (pinned-message :pinned-by)))
+                    message (assoc message :pinned pinned :pinned-by pinned-by)]
                 (merge message %)))
             %)
          message-list)))
 
 (defn albumize-messages
```

Comment:
This is the important change; the rest is code style fix applied automatically because for some reason it was broken